CMakeLists.txt

```diff
@@ -310,6 +321,7 @@ add_library(${TARGET}
     ${GGML_OPENCL_SOURCES}
     whisper.h
     whisper.cpp
+    ${OpenVINO_SOURCES}
```

Comment:
Use `OPENVINO_SOURCES`

However, why not make a separate target `whisper.openvino` similar to how `whisper.coreml` works?

Reply:
Let me try it again. I had originally tried to add it as a separate target and had some weird issues -- I intended to circle back though, so thanks for the reminder.

Reply:
okay, see latest commit ([76c4186](https://github.com/ggerganov/whisper.cpp/pull/1037/commits/76c41863933432b5f378c938cb27bacdc6b43912))

I added openvino-encoder to dedicated OBJECT target:
```
add_library(${TARGET} OBJECT
        openvino/whisper-openvino-encoder.h
        openvino/whisper-openvino-encoder.cpp
        )
```

And this target is linked to whisper just like coreml:
```
if (WHISPER_OPENVINO)
    target_link_libraries(${TARGET} PRIVATE whisper.openvino)
endif()
```

I was thinking of making it SHARED, but I think it'd be more of a hassle to have to carry around a separate .dll / .so..

This builds fine, and did some minimal testing on Windows 11 & Ubuntu.